apps/ergw_core/src/ergw_core.erl

```diff
@@ -77,16 +77,16 @@ is_running() ->
 
 %% get global PLMN Id (aka MCC/MNC)
 get_plmn_id() ->
-    [{config, plmn_id, #{mcc := MCC, mnc := MNC}}] = ets:lookup(?SERVER, plmn_id),
+    {ok, #{mcc := MCC, mnc := MNC}} = ergw_core_config:get([plmn_id], undefined),
```

Comment:
Looks like will be problem if will return `{ok, undefined}` default value

Reply:
yes, that is intended. If this function is called before ergw is properly configure, then it should crash.